features/retry_auto_attach.feature

```diff
@@ -117,19 +117,14 @@ Feature: auto-attach retries periodically on failures
 
     Examples: ubuntu release
       | release | machine_type  |
-      | xenial  | aws.generic   |
```

Comment:
auto-attach retries should work on aws, right? What changed?

Reply:
Sorry I should read commit messages before commenting :facepalm:

Reply:
Done